src/terminal/parser/OutputStateMachineEngine.hpp

```diff
@@ -238,7 +238,7 @@ namespace Microsoft::Console::VirtualTerminal
         bool _GetOscSetColor(const std::wstring_view string,
                              DWORD& rgb) const noexcept;
 
-        static constexpr DispatchTypes::CursorStyle DefaultCursorStyle = DispatchTypes::CursorStyle::BlinkingBlockDefault;
+        static constexpr DispatchTypes::CursorStyle DefaultCursorStyle = DispatchTypes::CursorStyle::BlinkingBlock;

```

Comment:
If you go with my suggested renaming of the enums, don't forget to change this to `UserDefault`, or whatever you decide for the name.